integration-cli/docker_cli_run_unix_test.go

```diff
@@ -299,7 +299,9 @@ func (s *DockerSuite) TestRunEchoStdoutWithMemoryLimit(c *check.C) {
 	}
 }
 
-// should run without memory swap
+// We set memory limit and disabled swap memory limit, this means the
```

Comment:
Oh you are right! Forgot this was unnecessary. I agree the name is rather explicit though :confused: